lib/jekyll/theme.rb

```diff
@@ -63,7 +63,8 @@ def realpath_for(folder)
       # However, symlinks are allowed to point to other directories within the theme.
       Jekyll.sanitized_path(root, File.realpath(Jekyll.sanitized_path(root, folder.to_s)))
     rescue Errno::ENOENT, Errno::EACCES, Errno::ELOOP
-      Jekyll.logger.warn "Invalid theme folder:", folder
+      Jekyll.logger.debug "Theme Meta:", "Directory #{folder.inspect} does not exist, or is " \
```

Comment:
Would just “Theme:” work here?

Reply:
IMO, **it wouldn't** because of the following output earlier in the program:
https://github.com/jekyll/jekyll/blob/e1ef5f9229f41785e48c2ee5344ced782f306c4d/lib/jekyll/theme.rb#L11-L12

To me, the following does not flow properly..
```
       Theme: my-jekyll-theme
Theme source: /path/to/theme-gem
       Theme: Directory "_includes" does not exist, or is..
```

Reply:
What about:

```sh
Theme: my-jekyll-theme
Theme source: /path/to/theme-gem
                          Directory "_includes" does not exist, or is..
```